plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/task/MetalavaSignature.kt

```diff
@@ -18,17 +51,27 @@ internal object MetalavaSignature : MetalavaTaskContainer() {
     ): TaskProvider<JavaExec> {
         return with(project) {
             tasks.register(name, JavaExec::class.java) {
+                require(extension.sourcePaths.isNotEmpty()) {"sourcePaths cannot be empty."}
```

Comment:
I'll get around to adding a linter one of these days!

```suggestion
                require(extension.sourcePaths.isNotEmpty()) { "sourcePaths cannot be empty." }
```